bidb/api/views.py

```diff
@@ -18,3 +24,38 @@ def submit(request):
         settings.SITE_URL,
         submission.buildinfo.get_absolute_url(),
     ), status=201 if created else 200)
+
+
+@require_http_methods(['GET'])
+def buildinfo_since(request, date):
+    try:
+        created = datetime.datetime.fromtimestamp(int(date) + 1)
+    except ValueError:
+        raise SuspiciousOperation("Date value should be an integer")
```

Comment:
I suggest returning `HTTPBadRequest` rather than `SuspiciousOperation` in this case:

```suggestion
        return HttpResponseBadRequest("Date value should be an integer")
```